tests/unit/test_datasets.py

```diff
@@ -74,12 +74,22 @@ def test_fetch(self, datasets):
         datasets.return_value.downloader.download.assert_called_once_with('file.xz')
 
     @patch('serenata_toolbox.datasets.Datasets')
-    def test_fetch(self, datasets):
-        fetch('file.xz', 'test')
-        datasets.assert_called_once_with('test')
-        datasets.return_value.downloader.download.assert_called_once_with('file.xz')
-
-    @patch('serenata_toolbox.datasets.Datasets')
     def test_fetch_latest_backup(self, datasets):
         fetch_latest_backup('test')
         self.assertTrue(datasets.return_value.downloader.download.called)
+
+    @patch('os.path.exists')
+    @patch('serenata_toolbox.datasets.Datasets')
+    def test_fetch_latest_backup_only_when_missing(self, datasets, os_path_exists):
+        datasets().downloader.LATEST = ('file1', 'file2', 'file3')
+        os_path_exists.side_effect = [False, True, False]
+        fetch_latest_backup('test')
+        datasets.return_value.downloader.download.assert_called_once_with(('file1', 'file3'))
+
+    @patch('os.path.exists')
+    @patch('serenata_toolbox.datasets.Datasets')
+    def test_fetch_latest_backup_with_force_all(self, datasets, os_path_exists):
```

Comment:
Shouldn't this test also be added to journey tests?

cc @cuducos @lipemorais

Reply:
@jtemporal since the dependencies are all mocked I believe that it's ok have this test under unit test folder. :+1: